pom.xml

```
@@ -286,6 +286,29 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```